sample-templates/query-assist-agent.json

```diff
@@ -0,0 +1,299 @@
+              "anthropic_version": "bedrock-2023-05-31"
+            }
+          }
+        },
+        {
+          "id": "register_claude_model",
+          "type": "register_remote_model",
+          "previous_node_inputs": {
+            "create_claude_connector": "connector_id"
+          },
+          "user_inputs": {
+            "name": "claude-instant",
+            "deploy": true
+          }
+        },
+        {
+          "id": "create_t2ppl_connector",
+          "type": "create_connector",
+          "user_inputs": {
+            "name": "sagemaker: t2ppl",
+            "description": "Test connector for Sagemaker t2ppl model",
+            "version": "1",
+            "protocol": "aws_sigv4",
+            "credential": {
+              "access_key": "PUT_YOUR_ACCESS_KEY_HERE",
+              "secret_key": "PUT_YOUR_SECRET_KEY_HERE"
+            },
+            "parameters": {
+              "region": "us-east-1",
+              "service_name": "sagemaker"
+            },
+            "actions": [
+              {
+                "action_type": "predict",
+                "method": "POST",
+                "headers": {
+                  "content-type": "application/json"
+                },
+                "url": "https://runtime.sagemaker.us-east-1.amazonaws.com/endpoints/PUT_YOUR_ENDPOINT_HERE/invocations",
+                "request_body": "{\"prompt\":\"${parameters.prompt}\"}"
+              }
+            ]
+          }
+        },
+        {
+          "id": "register_t2ppl_model",
+          "type": "register_remote_model",
+          "previous_node_inputs": {
+            "create_t2ppl_connector": "connector_id"
+          },
+          "user_inputs": {
+            "name": "t2ppl fine-tuned model",
+            "deploy": true
+          }
+        },
+        {
+          "id": "TransferQuestionToPPLAndExecuteTool",
+          "type": "create_tool",
+          "previous_node_inputs": {
+            "register_t2ppl_model": "model_id"
+          },
+          "user_inputs": {
+            "type": "PPLTool",
+            "name": "TransferQuestionToPPLAndExecuteTool",
+            "description": "Use this tool to transfer natural language to generate PPL and execute PPL to query inside. Use this tool after you know the index name, otherwise, call IndexRoutingTool first. The input parameters are: {index:IndexName, question:UserQuestion}",
+            "parameters": {
+              "response_filter": "$.completion",
+              "execute": false,
+              "prompt": "Below is an instruction that describes a task, paired with the index and corresponding fields that provides further context. Write a response that appropriately completes the request.\n\n### Instruction:\nI have an opensearch index with fields in the following. Now I have a question: ${indexInfo.question} Can you help me generate a PPL for that?\n\n### Index:\n${indexInfo.indexName}\n\n### Fields:\n${indexInfo.mappingInfo}\n\n### Response:\n"
```

Comment:
the claude prompt is here: https://github.com/opensearch-project/skills/pull/184/files#diff-778297aed573eb9f6ee67ce363e0159c095a9d6bd8ebebf8313aa90bae242249R2

once the PR is merged, this line can be changed to 
```suggestion
              "model_type": "claude"
```